client/app/libs/slim_lodash.js

```diff
@@ -0,0 +1,3 @@
+import bindAll from 'lodash/bindAll';
+import find from 'lodash/find';
+export default { bindAll, find };
```

Comment:
See previous discussion on the PR. The idea is to avoid including all of lodash when only specific functionality is required. I appreciate you taking time to look into this, thanks.